src/components/PlatformNavigator/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+import { Button, Flex, Text } from '@aws-amplify/ui-react';
+import { IconChevron } from '@/components/Icons';
+import { frameworks } from '@/constants/frameworks';
+import { InfoPopover } from './InfoPopover';
+import Link from 'next/link';
+
+export function PlatformNavigator({ currentPlatform }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const platformItem = frameworks.filter((platform) => {
+    return platform.title === currentPlatform;
+  })[0];
+
+  return (
+    <>
+      <nav aria-labelledby="platformBtn" className={`platform-navigator`}>
```

Comment:
Can we use a static aria-labelledBy for the nav? Otherwise screen readers will announce this as "Javascript" or something different if the platform changes. Maybe we can use an id on the <Text /> element below and reference that instead.